.storybook/styles.scss

```diff
@@ -8,3 +8,12 @@ body {
   // overwrite mc-sass-lib default body black background
   background-color: transparent !important;
 }
+
```

Comment:
mastersass defines these as white, so it was being inherited. Override for the styleguide.